.eslintrc.cjs

```diff
@@ -141,5 +141,9 @@ module.exports = {
         ],
       },
     },
+    {
+      files: 'cypress/**',
+      extends: 'plugin:cypress/recommended',
+    },
```

Comment:
I extended cypress eslint config for better developer experience

Reply:
<3